src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,62 @@
+package seedu.addressbook.ui;
+
+import java.io.PrintStream;
+
+/**
+ * Output formatter of TextUI class
+ * @author Ckjr
+ */
+public class Formatter {
+    
+    /** A decorative prefix added to the beginning of lines printed by AddressBook */
+    private static final String LINE_PREFIX = "|| ";
+    
+    /** A platform independent line separator. */
+    private static final String LS = System.lineSeparator();
+
+    private static final String DIVIDER = "===================================================";
+    
+    private final PrintStream out = System.out;
+    
+    public void print(String arg) {
+        showToUser(arg);
+    }
+    
+    public void printWithPrefix(String arg) {
```

Comment:
Formatter is doing 2 things now
1. formatting
2. printing
Essentially, you have made changes to violate SRP in a different way ;-)